Add info about Workflow API token in REST API (#2295)
astronomer/docs: astro/workspace-api-tokens.md

## Create a Workspace API token

> You can either create a new Workspace API token or add an existing Organization API token to a Workspace with appropriate permissions. You might want to add an Organization API token if your Organization API token only has Organization Member permissions, but you want to grant it Workspace Admin permissions for several Workspaces.

Reviewer: ```suggestion
There are two ways to use API tokens to interact with a Workspace:

- Create a new Workspace API token that's scoped only to a single Workspace.
- Add an existing [Organization API token](organization-api-tokens.md) to a Workspace and grant it Workspace permissions. You can add an Organization API token to multiple Workspaces.
```

   
6. Copy the token and store it in a safe place. You will not be able to retrieve this value from Astro again. 

### Add an Organization API token to a Workspace

> 

Reviewer: ```suggestion
You might want to add an Organization API token if your Organization API token only has Organization Member permissions, but you want to grant it Workspace Admin permissions for several Workspaces.

```

1. In the Cloud UI, open your Workspace.
   
2. Go to **Workspace Settings** > **Access Management** > **API Tokens**.
   
3. Click **API Token** > **Add Organization API Token**
   
4. Select the Organization API token from the drop-down and choose the appropriate access.

> 5. Click **Add**. A message appears on the bottom right corner "API token added".

Reviewer: ```suggestion
5. Click **Add**.
```

> 

Reviewer: ```suggestion
```

   
5. Optional. To delete a Workspace API token, click **Delete API Token**, enter `Delete`, and then click **Yes, Continue**.

## Delete or remove a Workspace API token

If you delete a Workspace API token or remove an Organization API token from your Workspace, make sure that no existing CI/CD pipelines are using the token. After you delete a Workspace API token, it can't be recovered. If you unintentionally delete an API token, create a new one and update any CI/CD workflows that used the deleted API token.

> 

Reviewer: ```suggestion
```

   
5. Optional. To delete a Workspace API token, click **Delete API Token**, enter `Delete`, and then click **Yes, Continue**.

## Delete or remove a Workspace API token

> If you delete a Workspace API token, make sure that no existing CI/CD pipelines are using it. After it's deleted, an API token cannot be recovered. If you unintentionally delete an API token, create a new one and update any CI/CD workflows that used the deleted API token.

Reviewer: ```suggestion
If you delete a Workspace API token or remove an Organization API token from your Workspace, make sure that no existing CI/CD pipelines are using the token. After you delete a Workspace API token, it can't be recovered. If you unintentionally delete an API token, create a new one and update any CI/CD workflows that used the deleted API token.
```

> If you are using an Organization API token for a Workspace, you can remove it's access from the Workspace. This API token will lose it's access to the given Workspace.

Reviewer: ```suggestion
```

1. In the Cloud UI, open your Workspace.
   
2. Go to **Workspace Settings** > **Access Management** > **API Tokens**.

3. Click **Edit** next to your API token.

> 4. Optional. If the token scope is Organiztion, then click **Remove API token**.
> 
> 5. Optional. If the token scope if Workspace, then click on **Delete API Token**, enter `Delete`, and then click **Yes, Continue**.

Reviewer: ```suggestion
4. If you're removing an Organization API token, click **Remove API token**. If you're deleting a Workspace API token, click **Delete API Token**, enter `Delete`, then click **Yes, Continue**.
```

## Rotate a Workspace API token

Rotating a Workspace API token lets you renew a token without needing to reconfigure its name, description, and permissions. You can also rotate a token if you lose your current token value and need it for additional workflows. 

When you rotate a Workspace API token, you receive a new valid token from Astro that can be used in your existing workflows. The previous token value becomes invalid and any workflows using those previous values stop working. 

> :::tip
> Please note that you can only rotate API tokens with Workspace scope from this menu.
> :::

Reviewer: ```suggestion
To rotate an Organization API token with Workspace permissions, see [Organization API tokens](organization-api-tokens.md).
```
